packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path';
+import { projectPaths, workspaceUtils } from '../../../src/';
+
+jest.mock('@salesforce/core', () => {
+  return {
+    Global: {
+      SFDX_STATE_FOLDER: '.sfdx'
+    }
+  };
+});
```

Comment:
_fancy_

Reply:
this was a bit of an adventure to figure out lol